docs/local-dev.md

```diff
@@ -0,0 +1,96 @@
+# Local Development of the calendar
+
+This file contains tips and tricks to help with local development of the
+calendar system. Please add to it as you find 'gotchas' or tips.
+
+## How does this all work?
+
+By moving to docker, your local development should run much more like
+production and dependencies (like setting up an LDAP endpoint) are handled
+automatically. `docker-compose.yml` (and it's cousin `docker-compose.prod.yml`)
+contain the definitions of our docker 'environment', allowing once
+docker-compose and docker are setup anyone to simply run `docker-compose up`
```

Comment:
Verbage is a little confusing to me, maybe add parenthesis or restructure?
allowing (once docker-compose and docker are set up) anyone to simply run `docker-compose up`
and start a local calendar system.

Reply:
Parens added.